src/materialized/tests/pgwire.rs

```diff
@@ -150,9 +150,11 @@ fn test_conn_params() -> Result<(), Box<dyn Error>> {
                 .get::<_, String>(0),
             "newdb",
         );
-        client.batch_execute("CREATE DATABASE newdb").await?;
         client
-            .batch_execute("CREATE MATERIALIZED VIEW v AS SELECT 1")
+            .batch_execute(
+                "CREATE DATABASE newdb; \
+                 CREATE MATERIALIZED VIEW v AS SELECT 1",
```

Comment:
(Since I know how much you love formatting consistency)

Either the last statement should end with a semicolon, or the last statement in the other `batch_execute` calls shouldn't.